backend/typescript/graphql/types/taskType.ts

```diff
@@ -0,0 +1,63 @@
+import { gql } from "apollo-server-express";
+
+const taskType = gql`
+  enum Status {
+    PENDING_APPROVAL
+    INCOMPLETE
+    COMPLETE
+    EXCUSED
+  }
+
+  enum Recurrence_Frequency {
+    DAILY
+    WEEKLY
+    BI_WEEKLY
+  }
+
+  input InputTaskDTO {
+    categoryId: Int!
+    title: String
+    status: Status
+    description: String
+    assigneeId: Int!
+    assignerId: Int!
+    creditValue: Int
+    startDate: DateTime!
+    endDate: DateTime
+    comments: String
+    recurrenceFrequency: Recurrence_Frequency
+  }
+
+  type TaskDTO {
+    id: Int!
+    category: String!
```

Comment:
Note for future: consider making category/assigner/assignee to return the entire object instead of individual fields